gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -96,6 +104,7 @@ public Kafka09DataWriter(Properties props) {
   public Kafka09DataWriter(Producer producer, Config config)
   {
     this.topic = config.getString(KafkaWriterConfigurationKeys.KAFKA_TOPIC);
+    provisionTopic(topic,config);
```

Comment:
to preserve backward compatibility, please make this an optional step with default set to false

Reply:
The default behaviour does create the topics if not present, the clustering provisioning will be active if writer.kafka.zookeeper property is defined in configuration.